dist/schema.json

```diff
@@ -13145,9 +13181,25 @@
               "follow_request",
               "favourite",
               "poll",
-              "update",
-              "admin.sign_up",
-              "admin.report"
+              "update"
+            ]
+          },
+          "alerts[admin": {
```

Comment:
`WebPushSubscription` is getting an property `alerts[admin`. That's malformed. It looks like it's taking `alerts[admin.sign_up]` and parsing it to `alerts[admin` -> `sign_up]` when it should be parsed to `alerts` -> `admin.sign_up`.